Content.Shared/_RMC14/Xenonids/Construction/Tunnel/XenoTunnelSystem.cs

```diff
@@ -822,6 +831,21 @@ private void SetEnabledStatusAllAbilities(EntityUid ent, bool newStatus)
         }
     }
 
+    private void DisableSpecificAbilities(EntityUid ent)
+    {
+        var actions = _action.GetActions(ent);
+        foreach (var action in actions)
+        {
+            var actionName = Name(action).ToLower();
+            if (actionName.Contains("rest") || actionName.Contains("regurgitate"))
+            {
+                _action.SetEnabled(action.AsNullable(), false);
+            }
+        }
```

Comment:
This should use RMCActionsSystem.GetActionsWithEvent, not compare the action name